client/src/App.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import PlayersListTable from "./components/PlayerListTable/PlayerListTable";
 import ClubList from "./components/ClubList/ClubList";
+import NavBar from "./components/NavBar/NavBar";
+import { Routes, Route } from "react-router-dom";
+import { url } from "./utils";
 
 const App = () => (
-    <>
-      <PlayersListTable />
-      <ClubList />
-    </>
-  );
-
+  <>
+    <NavBar />
+    <Routes>
+      <Route path={url + '/'} element={<PlayersListTable />} />
```

Comment:
```suggestion
      <Route path="/" element={<PlayersListTable />} />
```
this should be good enough, no?

Reply:
It works